src/System.Security.Principal.Windows/src/Microsoft/Win32/SafeHandles/SafeTokenHandle.cs

```diff
@@ -45,6 +43,14 @@ internal void InitialSetHandle(IntPtr h)
             base.handle = h;
```

Comment:
Is this (`InitialSetHandle`) even used?  Given that the handle will be valid after the constructor returns, I'm not sure we need it.